folding-schemes/src/folding/nova/mod.rs

```diff
@@ -176,7 +176,7 @@ where
             rW: C::ScalarField::zero(),
```

Comment:
Should the blinding factors of a new Nova `Witness` (`rE`, `rW`) be updated to support actual non-zero values?

(to something similar to what hypernova does for LCCCS & CCCS, eg. [sonobe/folding-schemes/src/folding/hypernova/lcccs.rs#L45](https://github.com/privacy-scaling-explorations/sonobe/blob/edadcdd520b8cf657cfa5c679dd09c6579759b0c/folding-schemes/src/folding/hypernova/lcccs.rs#L45) )

Reply:
Yes you are right! In fact, I updated the PR quite a bit to make sure that those blinding values are set correctly when running `prove_step` for both [nova](https://github.com/privacy-scaling-explorations/sonobe/blob/2707b0cf8ec03a5ca8946eee59b179bb4f12b498/folding-schemes/src/folding/nova/mod.rs#L469) and [hypernova](https://github.com/privacy-scaling-explorations/sonobe/blob/2707b0cf8ec03a5ca8946eee59b179bb4f12b498/folding-schemes/src/folding/hypernova/mod.rs#L511)